### ISSUE TITLE
Start calendar with current month

### ISSUE THREAD

**Issue by krizajb** [Paul-DS/bootstrap-year-calendar#18]
I didn't find an option to display calendar (12 months) starting with specific Date, current month.

**Comment by krizajb** [comment]
I made a simple workaround by display two calendars, first displays current year months and second months of next year. In next step I have simply hidden the past months of current year and hidden the months of next year. If month is odd number the first month of next year has to be moved (appended) to first year.

	var currentMonth = new Date().getMonth();
	var isOdd = isOdd(currentMonth);

	$('#first_calendar .month-container').each(function(idx, el){
    	if (idx < currentMonth) {
			$(this).css("display","none");
		}
	})

	if (isOdd) {
		$('#second_calendar .month-container').first().detach().appendTo("#first_calendar");
		currentMonth -=2;
	} else {
		currentMonth -=1;
	}

	$('#second_calendar .month-container').each(function(idx, el) {
    	if (idx > currentMonth) {
			$(this).css("display","none");
		}
	})

Don't forget to set startYear: new Date().getFullYear() and startYear: new Date().getFullYear()+1 on second calendar.

**Comment by Paul-DS** [comment]
You want to display something like that ?

![image](https://cloud.githubusercontent.com/assets/15329946/14798441/e3262718-0b37-11e6-8559-63bf907ac5a3.png)

Yes currently this is not managed, especially because the calendar is displayed for a given (unique) year. The HMI is not intended for that (current year displayed at the top, etc..)

**Comment by krizajb** [comment]
Yes, I needed that but as you can see I made a workaround, here is the solution (probably temporary address since it's not published yet, so I will update the url when so): http://www.shappa.si/koledar-new.html

**Comment by Paul-DS** [comment]
Ok super !

Given your example, I will add an option to remove the header. In this case it will not be present in the DOM (and not just hidden with CSS).

**Comment by Paul-DS** [comment]
I just added the option to display\hide the calendar header : `displayHeader`

**Comment by flash506** [comment]
this solution is ok for just viewing the calendar but when trying to select a date range between December and January it doesn't work...

**Comment by jmrp81** [comment]
Thanks, It is a magnificent calendar @Paul-DS and the solution of the months has helped me @krizajb

**Comment by Paul-DS** [comment]
I know this thread is pretty old, but I'm working on this feature in the new version of the library: [js-year-calendar](https://github.com/year-calendar/js-year-calendar)
You can follow the development (and test the functionality) in the PR year-calendar/js-year-calendar#32

**Comment by Paul-DS** [comment]
The feature is available in js-year-calendar@v2.0.0. Check the options `startDate` and `numberMonthsDisplayed` in the [documentation](https://year-calendar.github.io/js-year-calendar/documentation)
